frontend/src/helpers/colors/color.provider.ts

```diff
@@ -0,0 +1,62 @@
+import { VerboseDate } from "../../logic/real-schedule-logic/month.logic";
+import { WeekDay } from "../../state/models/schedule-data/month-info.model";
+import { ShiftCode } from "../../state/models/schedule-data/shift-info.model";
+import { CellColorSet } from "./cell-color-set.model";
+import { Colors } from "./colors";
+
+export class ColorProvider {
+  static get DEFAULT_COLOR_SET(): CellColorSet {
+    return { textColor: Colors.BLACK, backgroundColor: Colors.WHITE };
+  }
+
+  static getShiftColor(
```

Comment:
@Sig00rd 
Za pomocą tej metody możesz pokolorować odpowiednie zmiany. Opisałem to w komentarzu do zadania:
https://nurse-scheduling.atlassian.net/browse/NS-62?focusedCommentId=10008